apps/strapi-cms/Dockerfile

```
@@ -28,4 +28,5 @@ ENV PATH /opt/node_modules/.bin:$PATH
 RUN chown -R node:node /opt/app
 USER node
 EXPOSE 1337
+RUN npm run configrestore
 CMD npm run start
```

```suggestion
CMD npm run configrestore & npm run start
```

After reading a little about docker's CMD ([this SO question specifically](https://stackoverflow.com/questions/54244989/run-multiple-scripts-in-docker-image)) I would go for this solution.

Running the two commands in parallel should be fine since `configrestore` simply updates the DB and the admin panel generated by `start` will then read from it once launched by the user.

---

is the command `npm run configrestore` needed to be run also on local machine (e.g. during active development)?

---

Not necessarily.